### ISSUE TITLE
Add support for parsing with alternative field separators

### ISSUE THREAD

**Issue by da4089** [da4089/simplefix#15]
FIX uses the ASCII SOH character to separate (well, maybe terminate) fields.  But for human consumption, these are often replaced with, eg. `|` when logging messages.

Add optional support for parsing messages using non-standard separators.

**Comment by da4089** [comment]
Sorry, I don't understand what you're trying to do here.  Normally, pipes are used only for logging (so that humans can read the messages), but proper FIX protocol uses ASCII SOH characters.

Can you explain in a little more detail?  Or maybe a snippet of the code?

**Comment by zoakes** [comment]
Yeah I realized after sending this that it only uses the pipes in parse, not the actual encoded message. Testing again today, I’ll delete this if it runs as I expect it to — sorry about that!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
